admin-notice/admin-notice.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Plugin Name: Admin Notice
+ * Description: Adds dismisable notice in admin area
+ * Version:     0.1.0
+ * Author:      Automattic VIP
+ * Author URI:  https://wpvip.com
+ * License:     GPLv2 or later
+ *
+ * @package Automattic\VIP\Admin_Notice
+ */
```

Comment:
Why does these plugin headers need to be duplicated here?

Reply:
Oh I just thought that is the usual way of doing it when looking into the other plugins. But keeping it in one place is definitely better. I removed it.